MMM-Spotify.js

```diff
@@ -511,15 +511,21 @@ Module.register("MMM-Spotify", {
     var img_url
     var display_name
     if (playbackItem.album){
-      img_url = playbackItem.album.images[img_index].url
+      if (playbackItem.album.images.lengh > 0) {
+        img_url = playbackItem.album.images[img_index].url
+      }
       display_name = playbackItem.album.name
     }
     else{
-      img_url = playbackItem.images[img_index].url
+      if (playbackItem.images.length > 0) {
+        img_url = playbackItem.images[img_index].url
+      }
       display_name = playbackItem.show.name
     }
 
-    this.updateCoverImage(img_url)
+    if (img_url) {
```

Comment:
Thank you for this useful PR @sidevesh 
One thought about it though: if you switch from a song with an existing image to one without, the image from the previous song would still be displayed, because `updateCoverImage` would not have been called.

I would suggest to adapt this to always call the `updateCoverImage` function but then put some other image.

In fact, the function itself already has a fallback for this:
https://github.com/skuethe/MMM-Spotify/blob/1ffbec7109ad912ea0bbe161fc0eb942d4cd22be/MMM-Spotify.js#L554-L557

So either we add a "no cover image available" image or just always call `this.updateCoverImage(img_url)`.